### PR TITLE
feat(dependencies) Add Laravel 6 support

### PR DESCRIPTION
**Pull request by pablo-bossi**
Update dependencies and tests for Laravel 6 support

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
         ]
     },
     "require": {
-        "laravel/framework": "~5.3"
+        "laravel/framework": "~6.0"
```
Comment (pablo-bossi) [inline composer.json:14]: Support for Laravel 6

File: composer.json [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev",
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
+        "php-coveralls/php-coveralls": "2.2.*",
```
Comment (pablo-bossi) [inline composer.json:20]: satooshi/php-coveralls is depreceted so replaced by the suggested replacement which worked out fine

File: composer.json [hunk 3]
```diff
@@ -11,13 +11,13 @@
...
+        "laravel/framework": "~6.0"
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev",
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
```
Comment (pablo-bossi) [inline composer.json:19]: Update dependencies to meet Laravel 6 needs

File: phpunit.xml [hunk 4]
```diff
@@ -7,8 +7,7 @@
      convertNoticesToExceptions="true"
      convertWarningsToExceptions="true"
      processIsolation="false"
-     stopOnFailure="false"
-     syntaxCheck="false">
```
Comment (pablo-bossi) [inline phpunit.xml:11]: syntaxCheck is unrecognized on phpUnit 8

File: tests/ExceptionHandlerTest.php [hunk 5]
```diff
@@ -26,7 +26,7 @@ public function format(JsonResponse $response, Exception $e, array $reporterResp
 
 class ExceptionHandlerTest extends TestCase {
 
-    public function setUp()
+    public function setUp() :void
```
Comment (pablo-bossi) [inline tests/ExceptionHandlerTest.php:29]: Match declaration of TestCase with PHPUnit 8

File: tests/ExceptionHandlerTest.php [hunk 6]
```diff
@@ -124,10 +124,9 @@ public function testReportInvalidReporterClass()
 
         $property->setValue($handler, $config);
 
-        $this->setExpectedException(
-            \InvalidArgumentException::class,
-            'invalid: stdClass is not a valid reporter class.'
-        );
+        $this->expectException(\InvalidArgumentException::class);
+
+        $this->expectExceptionMessage('invalid: stdClass is not a valid reporter class.');
```
Comment (pablo-bossi) [inline tests/ExceptionHandlerTest.php:129]: setExpectedException is no longer supported on phpUnit 8, so replaced by the equivalent code

File: tests/ExceptionHandlerTest.php [hunk 7]
```diff
@@ -156,13 +155,11 @@ public function testInvalidFormatterClass()
 
         $property->setValue($handler, $config);
 
-        $this->setExpectedException(
-            \InvalidArgumentException::class,
-            sprintf(
-                "% is not a valid formatter class.",
-                get_class($formatter)
-            )
-        );
```
Comment (pablo-bossi) [inline tests/ExceptionHandlerTest.php:165]: setExpectedException is no longer supported in phpUnit 8, so replaced by equivalent code

File: tests/Formatters/ExceptionFormatterTest.php [hunk 8]
```diff
@@ -7,7 +7,7 @@
 class ExceptionFormatterTest extends TestCase
 {
 
-    public function setUp()
+    public function setUp() :void
```
Comment (pablo-bossi) [inline tests/Formatters/ExceptionFormatterTest.php:10]: Match setUp declaration for phpUnit 8

File: tests/Reporters/BugsnagReporterTest.php [hunk 9]
```diff
@@ -11,16 +11,16 @@ class BugsnagReporterTest extends TestCase
 
     protected $client;
 
-    public function setUp()
+    public function setUp() :void
```
Comment (pablo-bossi) [inline tests/Reporters/BugsnagReporterTest.php:14]: Match setUp declaration for phpUnit 8

**Comment by esbenp** [comment]
Thanks @pablo-bossi !

**Pull request merged**